### ISSUE TITLE
Unfair experimental settings and performance without training tricks?

### ISSUE THREAD

**Issue by zhijiew** [JDAI-CV/FADA#5]
Hi, thank you for your great work. I have some questions about the your experiment details.

In your experiments, your settings include large batchsize (8), distributed training, sync batch normalization and many data argumentation techniques (including changing brightness, contrast, saturation and hue). But in the other experiments you compare (ResNet101 part of Table 3), their training methods are mostly very ’plain‘, they mostly use small batchsize (1), single-card training, and barely data augmentation. The training tricks you used usually can bring huge boost to your final performance. I think a comparison in fair settings will show the real improvement of your method.

**Comment by krumo** [comment]
Hi, thanks for your interest! It's a good question. Here is my opinion:

1. Some well-known tricks don't work for cross-domain settings. We all know a rational large batch size would benefit supervised learning without domain shift. However, when I apply distributed training and a large batch size for cross-domain semantic segmentation task, I don't observe an obvious performance improvement. Plus, when we do adversarial training, the statistics information for batchnorm is fixed, which would make a large batch size contribute little to the final performance. Here we use distributed training and a large batch size mainly because we want to increase the training speed instead of aiming to increase the final performance.

2. Actually different works utilize slightly different training settings and test settings. It would be difficult for us to re-train all compared methods under the exactly same setting for a complete comparison. Color augmentation is a very common trick for computer vision tasks and we think it is fair to incorporate it to our training pipeline. According to our experience, it could bring about 1% mIoU improvement. I think the color augmentation would not affect the conclusions in the paper.
